src/mlpack/config.hpp

```diff
@@ -73,41 +50,12 @@
 #endif
 
 //
-// Perform autodetection of STB if possible.
+// STB is part of mlpack now, if the user does not want to use the integrated
```

Comment:
```suggestion
// STB is part of mlpack. If the user does not want to use the integrated
```
We probably won't touch this again for five years, so the comment will make it seem like the change just happened but we will be half a decade down the road :smile: